assets/arc-0015/main.py

```diff
@@ -0,0 +1,119 @@
+import algosdk
+import base64
+from pyteal import *
```

Comment:
For the purposes of clarity, I recommend not using `*` imports here. It's easier for the reader to discern where a particular python object is sourced in that case.